lego_manager/lego_manager.py

```diff
@@ -15,53 +16,119 @@ class LegoManager(rpyc.Service):
     __slots__ = ("_allocations", "_bg_threads")
     ALIASES = ["LegoManager"]
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: str, **kwargs: str) -> None:
         super().__init__(*args, **kwargs)
-        self._allocations = dict()
-        self._bg_threads = dict()
+        self._allocations: Dict = dict()
```

Comment:
Why variable type?

Reply:
Why not?

Reply:
It is not pythonic

Reply:
Fixed

Reply:
MyPy (python static typing checker) recommend to do so:
![image](https://user-images.githubusercontent.com/27344500/79685114-cc932500-823e-11ea-8c36-14598b2e015a.png)